molecule/util.py

```diff
@@ -95,6 +135,8 @@ def run_command(cmd, debug=False):
     :param debug: An optional bool to toggle debug output.
     :return: ``sh`` object
     """
+    if 'MOLECULE_LIMITED_SHEBANG' in os.environ:
```

Comment:
maybe booleanize this?

`to_bool(os.getenv('MOLECULE_LIMITED_SHEBANG', False))`